packethardware/utils.py

```diff
@@ -166,10 +166,10 @@ def get_supermicro_serial_port_settings(self):
             name = setting.get("name")
             selected_option = setting.get("selectedOption")
 
-            if selected_option == "Auto":
-                option_value_1 = setting.find(".//Option[@value='1']")
-                if option_value_1 is not None:
-                    selected_option = f"Auto ({option_value_1.text})"
+        if selected_option == "Auto":
+            option_value_1 = setting.find(".//Option[@value='1']")
+            if option_value_1 is not None:
+                selected_option = "Auto (" + option_value_1.text + ")"
```

Comment:
```suggestion
                selected_option = "Auto (%s)" % option_value_1.text
```
is cleaner I think. Shouldn't be in the habit of building up strings with `+`.

Reply:
The de-indent was unintentional, fixed that. Split into three commits with better descriptions for each change